homeassistant/data_entry_flow.py

```diff
@@ -165,6 +166,10 @@ class FlowResult(TypedDict, total=False):
     url: str
 
 
+class FlowResult(BaseFlowResult[str], total=False):
+    """Typed result dict."""
```

Comment:
As mentioned for `AuthFlowResult`, this could also be a type alias. Probably doesn't make sense though, does it?

Reply:
Can we just add a default to the TypeVar instead and remove the `BaseFlowResult`?

Reply:
Yes. That should work if you don't need / want to add any additional keys just for `FlowResult`.